justfile

```diff
@@ -114,8 +114,13 @@ uninstall-client: uninstall-client-index uninstall-client-application
 
 # Create a public `index.html` file.
 install-client-index:
+	#!/bin/sh
 	cp source/client/src/index.html public/static/index.html
-	sed -i '' "s@{MAP-PLACEHOLDER.svg}@`cat public/static/image/map-placeholder.svg | sed -E 's/^ +//g; s/"/'"'"'/g; s/</%3c/g; s/>/%3e/g; s/\#/%23/g' | tr -d "\n"`@" public/static/index.html
+	if [[ "$OSTYPE" == "darwin"* ]]; then
+		sed -i '' "s@{MAP-PLACEHOLDER.svg}@`cat public/static/image/map-placeholder.svg | sed -E 's/^ +//g; s/"/'"'"'/g; s/</%3c/g; s/>/%3e/g; s/\#/%23/g' | tr -d "\n"`@" public/static/index.html
```

Comment:
To avoid duplicating the entire command line, you can use `OSTYPE` to create a `SED_ARGUMENTS` variable set to `-i ''` for macOS, otherwise set to empty; and use it in the the command line.

Reply:
I try to put sed in place option as a just variable. such that the line can be shorter, and the command is also clear to see the difference between MacOS and Linux.  
Is this way acceptable?
Thanks for your review.

Reply:
Let's say yes!